recipe.cake

```diff
@@ -109,7 +109,7 @@ Task("Prepare-Chocolatey-Packages")
     // Run Chocolatey Unpackself
     CopyFile(BuildParameters.Paths.Directories.PublishedApplications + "/choco_merged/choco.exe", BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe");
 
-    StartProcess(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe", new ProcessSettings{ Arguments = "unpackself -f --allow-unofficial-build" });
+    StartProcess(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe", new ProcessSettings{ Arguments = "unpackself -f -y --allow-unofficial-build" });
```

Comment:
This is does not work to prevent the admin required popup for me:
![image](https://user-images.githubusercontent.com/38865330/175372422-4d99ad56-00dd-4a5d-9a11-e765519f60ee.png)

Reply:
It doesn't prevent it, but it does prevent it from blocking. Without the `-y` it would just sit there waiting until you provide a response.

Reply:
I have taken this for a spin, and it works as expected for me.